msbuild/Xamarin.MacDev.Tasks/Tasks/GetFileSystemEntries.cs

```diff
@@ -0,0 +1,7 @@
+namespace Xamarin.MacDev.Tasks
+{
+	public class GetFileSystemEntries : GetFileSystemEntriesTaskBase
+	{
```

Comment:
This needs to do something like this to actually execute the task remotely: https://github.com/xamarin/xamarin-macios/blob/main/msbuild/Xamarin.iOS.Tasks/Tasks/ACTool.cs#L10-L13 :)

Reply:
Fixed.